modAion/src/org/aion/zero/core/IRewardsCalculator.java

```diff
@@ -22,7 +22,7 @@
  *     Aion foundation.
  *
  ******************************************************************************/
-package org.aion.mcf.core;
+package org.aion.zero.core;
```

Comment:
Correct me if i'm wrong, but block rewards are not PoW-spefic, but rather a cornerstone of public blockchains as they are the incentive for people to participate in the network. So this piece of code belongs in the general case.

Reply:
@centrys-paul block rewards are not always necessary in permissioned/private/enterprise implementations - if we are talking about the bigger picture

Reply:
Ah, I see.  I had a similar thought as @centrys-paul -- since it is useful for use cases more general than PoW or Aion (although not necessarily always used as @centrys-cristian pointed out), it seemed like it would make sense for it to belong in the general case (modMcf).

I don't think it will have much practical impact right now where this class lives.  It seems we have a modAion for the Aion case, and then modMcf for the "general" case, but nothing in between.  modMcf seemed slightly more intuitive to me, but I'm fine with either one -- I don't have strong feelings about it.